ServerCore/Pages/Puzzles/Play.cshtml

```diff
@@ -237,6 +237,13 @@ else
     }
     else
     {
+        <br />
+        <a asp-page="/Events/FastestSolves"
+           asp-route-teamPuzzleSort="CountDescending"
+           asp-route-stateFilter="Unsolved">
+
+            Feeling stuck?
+        </a>
```

Comment:
Should this have the rest of the text? (line 77 of this file)

Reply:
Fixed in newest iteration
![image](https://github.com/PuzzleServer/mainpuzzleserver/assets/4121745/540005a6-2796-4974-88fb-d0fb4a70d4e9)